components/prow/cluster.yaml

```diff
@@ -10,5 +10,6 @@ cluster:
   - name: prow
     initialnodecount: 1
```

Comment:
Since `prow` node-pool will deploy

- prow
- prowjobs
- prometheus-meta
- nginx-ingress-pods
- grafana
- alert-manager

`initialnodecount` should be atleast 4

Reply:
why? we run all of these on the same node, The machine is big enough to handle the load.

Reply:
none of these should cause a big load.

Reply:
Since all these components have `nodeSelector: cloud.google.com/gke-nodepool: prow`
I think for resilience, there should be at-least 2 nodes.
And also, if nodecount is less, we should add resource.limits so that a pod does'nt consume more resource and harm other pods.

Reply:
that is easy to scale if needed, lets keep it to a minimum and if facing issues will revisit.

Reply:
it is difficult to know what we need before running some tests so lets test and than adjust as needed. I don't like wasting resources until we are sure we need this. It might happen that it only consumes 10% of CPU and memory.